src/components/Theme.js

```diff
@@ -46,20 +46,20 @@ type Theme = {
 
 const theme: Theme = {
     palette: {
-        primary: "#81f1f7",
+        primary: "#39c3ef",
```

Comment:
I think I prefer the old color, but we can ask the team on Monday what they think.